std/os/bits/linux/arm64.zig

```diff
@@ -0,0 +1,405 @@
+// arm64-specific declarations that are intended to be imported into the POSIX namespace.
```

Comment:
kinda annoying that github doesn't recognize renames the way git does. (and yes i see you added new syscalls)

Reply:
I don't think I added new syscalls. I don't remember doing that, and I just did a diff with this file and the corresponding one in master branch. I only tried to fix rename/renameat & friends.